entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java

```diff
@@ -167,6 +169,20 @@ public EntitySite(String title,
         getUserRoles(); // populate the user roles
     }
 
+    public EntitySite(Site site, boolean includeGroups, boolean justInfo) {
```

Comment:
Right, my point though is if it's not used, why include it? If the only reason is to create a new method signature, there should at least be some javadoc explaining this.